crates/ra_assists/src/handlers/add_custom_impl.rs

```diff
@@ -52,7 +52,7 @@ pub(crate) fn add_custom_impl(acc: &mut Assists, ctx: &AssistContext) -> Option<
         format!("Add custom impl `{}` for `{}`", trait_token.text().as_str(), annotated_name);
 
     let target = attr.syntax().text_range();
-    acc.add(AssistId("add_custom_impl"), label, target, |builder| {
+    acc.add(AssistId("add_custom_impl"), AssistKind::Refactor, label, target, |builder| {
```

Comment:
Wdyt, does it make sense to move `AssistKind` into `AssistId` struct? Kind is kind-of determined by ID.

Reply:
On the other hand moving it into the `AssistId` struct makes it seem like `AssistId("add_custom_impl", AssistKind::Refactor)` and `AssistId("add_custom_impl", AssistKind::RefactorExtract)` are two separate assists.

Reply:
@matklad I thought about this and almost made that change If that makes more sense I can do that.

@bjorn3 Aren't the id's supposed to be unique?